monkestation/code/modules/bloodsuckers/structures/bloodsucker_crypt.dm

```diff
@@ -130,6 +130,8 @@
 	var/disloyalty_confirm = FALSE
 	/// Prevents popup spam.
 	var/disloyalty_offered = FALSE
+	// Prevent spamming torture via spam click. Otherwise they're able to lose a lot of blood quickly
+	var/torture_debounce = FALSE
```

Comment:
im confused as to why the var is named this

Reply:
https://en.wiktionary.org/wiki/debounce

basically, it's the name in a few languages and mechanical engineering for what it is doing (preventing it from repeatedly triggering)

Reply:
https://en.wiktionary.org/wiki/debounce

it's a term in certain languages basically describing what the variable does. I'll change the name of this to something else.